CHANGELOG.md

```diff
@@ -3,6 +3,29 @@ Changes by Version
 
 <!-- next version -->
 
+## 0.76.1
```

Comment:
there is missing section for all components in this release

Reply:
What do you mean by missing section?